tests/runtime/json-output

```diff
@@ -156,12 +156,12 @@ AFTER ./testprogs/simple_struct
 
 NAME print_avg_map_args
 RUN {{BPFTRACE}} -q -f json -e 'BEGIN { @["a"] = avg(10); @["b"] = avg(20); @["c"] = avg(30); @["d"] = avg(40); print(@, 2, 10); clear(@); exit(); }'
-EXPECT {"type": "stats", "data": {"@": {"c": 3, "d": 4}}}
```

Comment:
This would be a breaking change so I can change it back to stats. I guess I wasn't sure if 'avg' should really be considered 'stats' even though min/min are not.